test/unit/base-plugin-test.js

```diff
@@ -408,7 +409,7 @@ describe('base plugin', function() {
     }
 
     function addEvent(event, node, plugin) {
-      events.push([event, getId(node), plugin.config]);
+      events.push([event, getId(node), determineRuleConfig(plugin.config).config]);
```

Comment:
Can you beef this up to include the severity, then update the assertions below to show both the config and the severity.